ebos/ecltransmissibility_impl.hh

```diff
@@ -495,7 +500,7 @@ update(bool global, const std::function<unsigned int(unsigned int)>& map, const
     // Loop over all elements (global grid) and store Cartesian index
     for (const auto& elem : elements(grid_.leafGridView())) {
         int elemIdx = elemMapper.index(elem);
-        int cartElemIdx = cartMapper_.cartesianIndex(elemIdx);
+        int cartElemIdx =  this->lookUpCartesianData_.template getFieldPropCartesianIdx<Grid>(elemIdx);
```

Comment:
Can you remind me what the difference between the calls is?
I was under the impression that both should do the same. In that case we should probably use the old version.

Reply:
They do the same with and without LGRs, when all the field properties are assumed to be given in the level 0/coarse grid. The difference is when field properties for LGRs are given in their respected refined grids (levels>0). Then, the lookUpCartesianData_ method will get the "local/level" Cartesian index of the leaf grid view cell, on the level this cell was born, and this is not necessarily level 0.

With the current assumptions, we can of course leave the old line of code, and go back to change it when refined field properties will be supported.

Reply:
it's back to the old version